#!/usr/bin/env python
PACKAGE = "IP_SAUVC"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("a1min_param",    int_t,    0, "a1min parameter", 0,  0, 260)
gen.add("a1max_param",    int_t,    0, "a1max parameter", 0,  0, 260)
gen.add("a2min_param",    int_t,    0, "a2min parameter", 0,  0, 260)
gen.add("a2max_param",    int_t,    0, "a2max parameter", 0,  0, 260)
gen.add("a3min_param",    int_t,    0, "a3min parameter", 100,  0, 260)
gen.add("a3max_param",    int_t,    0, "a3max parameter", 255,  0, 260)

exit(gen.generate(PACKAGE, "IP_SAUVC", "pinger"))
